ngraph_bridge/enable_variable_ops/ngraph_replace_variable_modifiers.cc

```diff
@@ -35,6 +35,28 @@ namespace ngraph_bridge {
 // 1. ReplaceModifiers
 // 2. RewriteForTracking
 
+Status CreateBinaryOpNode(NodeBuilder::NodeOut* ndef_gen_op,
+                          std::string new_name_op, std::string op_type,
+                          NodeBuilder::NodeOut input1, NodeBuilder::NodeOut input2, DataType dtype, Node*& node,
+                          Graph*& graph, bool replace_input)
+                           {
+  Node* generic_op;
+
+  TF_RETURN_IF_ERROR(NodeBuilder(new_name_op, op_type)
+                         .Input(input1)
+                         .Input(input2)
+                         .Attr("T", dtype)
+                         .Device(node->assigned_device_name())
+                         .Finalize(graph, &(generic_op)));
+  generic_op->set_assigned_device_name(node->assigned_device_name());
+
+  if (replace_input) {
```

Comment:
Replacing control edges could be done outside this utility function call. We can use this utility to just create the node.
Try this to get the node from the NodeOut structure
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/graph/node_builder.h#L64
Moreover we can just pass the string that corresponds to (node-> assigned_device_name()) instead of the node itself.